addons/website_sale/migrations/10.0.1.0/post-migration.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    pl_model = env['product.pricelist']
+    sql = """select pricelist_id, website_id, selectable from
+          website_pricelist_openupgrade_10"""
+    cr.execute(sql)
+    pricelists = []
+    for pricelist_id, website_id, selectable in cr.fetchall():
+        pl_model.search([('id', '=', pricelist_id)]).write({
```

Comment:
this query does not consider those pricelists that need to be set at the current default website, so anyways we are just copying the old column so we can save the first loop with the same query , but we still need the second to update all those possible pricelist ids not in the website_pricelist_openupgrade_10 column with the default website.   addressed in 20c76159af016f73e7b271727ae9bba0cfb962ba  waiting for checks to pass.